stable/sentry/values.yaml

```diff
@@ -14,7 +14,7 @@ web:
   replicacount: 1
   resources:
     limits:
-      cpu: 500m
+      cpu: 4000m
```

Comment:
Is it too big limit, isn't it?

Reply:
as sentry [stated](https://docs.sentry.io/server/installation/), sentry is not designed to run on low resource systems. we faced too many container crashes on our productions systems with the previous limit which was resolved by increasing the resource limits

Reply:
I'm sorry you faced the crashes.
But your requirements pushes user to have at least 2 x c5.2xlarge (for worker and web).
To be honest I'd like not to specify default resources.

Reply:
you mean it's ok to remove them completely? or just keep the old low limitations?
also does setting `limits` (and not `requests`) makes user to have the amount of resources that are stated on `limites` section?

Reply:
It doesn't, but:
```When Pods use more memory than their limit, a process that is using the most amount of memory, inside one of the pod's containers, will be killed by the kernel```
https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/resource-qos.md#qos-classes

I think we should save old values, I'll rework it later.

Reply:
the point is sentry seems useless with that low amount of resources. why should we have them as default and ship a boken chart to the users?

Reply:
I wouldn't say it's broken.
For example, I'm using it with default resources and it works properly.
https://imgur.com/a/hy9hY2C
https://imgur.com/a/AMFtHFW

But if you're totally sure we shouldn't use the limits were specified before then please remove them at all and set:
```
resources: {}
  # We usually recommend not to specify default resources and to leave this as a conscious
  # choice for the user. This also increases chances charts run on environments with little
  # resources, such as Minikube. If you do want to specify resources, uncomment the following
  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
  # limits:
  #  cpu: 100m
  #  memory: 128Mi
  # requests:
  #  cpu: 100m
  #  memory: 128Mi
```

Reply:
I've changed it to empty resources